src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace src\oop\app\src\Parsers;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class KinoukrDomCrawlerParserAdapter implements ParserInterface
+{
+    private $crawler;
+
+    public function __construct()
+    {
+        $this->crawler = new Crawler();
```

Comment:
please do not use new keyword, use dependency injection:
public function __construct(Crawler $crawler)
{
   $this->crawler = $crawler;